Windows/SettingsWindow.xaml

```diff
@@ -20,192 +20,257 @@
+                                <Slider x:Name="sliderSongVol" Maximum="1" ValueChanged="SliderSongVol_ValueChanged" IsMoveToPointEnabled="True" MouseLeftButtonUp="sliderSongVol_MouseLeftButtonUp" Thumb.DragCompleted="sliderSongVol_DragCompleted"/>
+                            </DockPanel>
+                        </Border>
+
+                        <Border Grid.Row="5" Grid.Column="0">
+                            <TextBlock FontSize="10.5" VerticalAlignment="Center">Default Note Volume</TextBlock>
+                        </Border>
+
+                        <Border Grid.Row="5" Grid.Column="1">
+                            <DockPanel>
+                                <TextBlock x:Name="txtDrumVol" DockPanel.Dock="Right" Width="30" Margin="5 0 0 0">0%</TextBlock>
+                                <Slider x:Name="sliderDrumVol" Maximum="1" ValueChanged="SliderDrumVol_ValueChanged" IsMoveToPointEnabled="True" MouseLeftButtonUp="sliderDrumVol_MouseLeftButtonUp" Thumb.DragCompleted="sliderDrumVol_DragCompleted"/>
+                            </DockPanel>
+                        </Border>
+                    </Grid>
+                    <Label x:Name="spectrogramOptionsLabel" Margin="0 10 0 0" Padding="10 5" FontWeight="Bold" FontSize="16" FontFamily="Bahnschrift" Visibility="Collapsed">Spectrogram</Label>
+                    <Grid x:Name="spectrogramOptions" Margin="15 0 5 0" VerticalAlignment="Center" Visibility="Collapsed">
+                        <Grid.Resources>
+                            <Style TargetType="Border">
+                                <Setter Property="Padding" Value="5,5,5,5" />
+                            </Style>
+                        </Grid.Resources>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="*" />
+                            <RowDefinition Height="*" />
+                            <RowDefinition Height="*" />
+                            <RowDefinition Height="*" />
+                            <RowDefinition Height="*" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="110" />
+                            <ColumnDefinition Width="110" />
+                        </Grid.ColumnDefinitions>
+
+                        <Border Grid.Row="0" Grid.Column="0">
+                            <TextBlock VerticalAlignment="Center">Cache Images</TextBlock>
+                        </Border>
+
+                        <Border Grid.Row="0" Grid.Column="1">
+                            <CheckBox x:Name="checkSpectrogramCache" Click="checkSpectrogramCache_Click" />
+                        </Border>
+
+                        <TextBlock Grid.Row="1" Grid.ColumnSpan="2" FontSize="8" VerticalAlignment="Center" TextWrapping="WrapWithOverflow">
+                            Faster spectrogram loading after it's been generated once. Warning:
```

Comment:
I've discussed having this kind of warning with Waatsu, as he tends to package his maps outside of Edda's Export Map function and this is also shown in his mapping tutorial (it's quite outdated at the moment).

If you have any good idea about a place to put the cached images without risking an impact of those being included in the ZIP files by mistake, I'll gladly change the save location and remove this warning.

Reply:
We could put them in a temp folder in AppData, but we already store backups in the map folder so adding a few more images isn't really going to change anything.

Maybe we could move backups out of the map folder in future and the spectrograms could go with them.